samples/highcharts/demo/combo-timeline/demo.js

```diff
@@ -184,12 +204,87 @@ function onChartLoad() {
     }
 }
 
+// Get the turnover. Read the table from the HTML, sort by the joined/left
+// events and keep track of the number of employees.
+function getTurnover() {
+    let employees = 0;
+    return [].reduce.call(
+        document.getElementById('turnover').querySelectorAll('tr'),
+        (turnover, tr) => {
+            const dateJoined = Date.parse(tr.children[1].textContent);
+            if (!isNaN(dateJoined)) {
+                turnover.push({
+                    x: dateJoined,
+                    name: `${tr.children[0].textContent} joined`,
+                    accumulate: 1,
+                    image: tr.children[3].textContent || null
+                });
+            }
+
+            const dateLeft = Date.parse(tr.children[2].textContent);
+            if (!isNaN(dateLeft)) {
+                turnover.push({
+                    x: dateLeft,
+                    name: `${tr.children[0].textContent} left`,
+                    accumulate: -1,
+                    image: tr.children[3].textContent || null
+                });
+            }
+
+            return turnover;
+        },
+        [])
+        .sort((a, b) => a.x - b.x)
+        .map(event => Object.assign(
+            event, {
+                y: (employees += event.accumulate)
+            }
+        ));
+}
+
+const moreEmployees = [
```

Comment:
Looks like employees before 2019 are loaded from the HTML table then this is added for employees after 2019? That's not consistent. Add the new ones to the HTML table instead, if that is how we did it from before.